code/game/objects/items/stacks/sheets/mineral.dm

```diff
@@ -465,3 +465,91 @@ GLOBAL_LIST_INIT(abductor_recipes, list ( \
 
 /obj/item/stack/sheet/mineral/coal/ten
 	amount = 10
+
+/*
+ * Morphium
+ */
+
+/obj/item/stack/sheet/mineral/morphium
+	name = "Morphium"
+	desc = "A powerful material that mimics what the gray's use."
+	icon_state = "sheet-wavy"
+	item_state = "mhydrogen"
+
+/*
+ * Valhollide
+ */
+
+/obj/item/stack/sheet/mineral/valhollide
+	name = "Valhollide"
+	desc = "A powerful material made out of mashing particles into one another, ya science!"
```

Comment:
should probably be put to "A powerful material made out of mashing particles into one another. Yay science!"

Reply:
This is a movie ref

Reply:
oh ok